From ltsi-dev-bounces@lists.linuxfoundation.org Sat May 11 14:26:47 2013
From: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
Date: Sun, 12 May 2013 01:26:55 +0400
Subject: ARM: mach-shmobile: marzen: add SATA support
To: ltsi-dev@lists.linuxfoundation.org
Cc: vladimir.barinov@cogentembedded.com
Message-ID: <201305120126.55306.sergei.shtylyov@cogentembedded.com>


From: Vladimir Barinov <vladimir.barinov@cogentembedded.com>

Add SATA support to marzen_defconfig.

Signed-off-by: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
Signed-off-by: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit db30abdf3c07c564d4af0f991bb7f38b728f9035)

---
This patch (already merged by Linus) is intended for the Marzen patch series.

Changes from the upstream version:
- resolved reject due to missing interim patches.

 arch/arm/configs/marzen_defconfig |    4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm/configs/marzen_defconfig
+++ b/arch/arm/configs/marzen_defconfig
@@ -47,6 +47,10 @@ CONFIG_DEVTMPFS_MOUNT=y
 # CONFIG_STANDALONE is not set
 # CONFIG_PREVENT_FIRMWARE_BUILD is not set
 # CONFIG_FW_LOADER is not set
+CONFIG_ATA=y
+CONFIG_ATA_SFF=y
+CONFIG_ATA_BMDMA=y
+CONFIG_SATA_RCAR=y
 CONFIG_NETDEVICES=y
 # CONFIG_NET_VENDOR_BROADCOM is not set
 # CONFIG_NET_VENDOR_FARADAY is not set
